packages/components/src/tooltip/test/index.tsx

```diff
@@ -436,4 +436,50 @@ describe( 'Tooltip', () => {
 			await waitExpectTooltipToHide();
 		} );
 	} );
+
+	describe( 'nested', () => {
+		it( 'should render the outer tooltip and ignore inner tooltips', async () => {
+			render(
+				<Tooltip text="Outer tooltip">
+					<Tooltip text="Middle tooltip">
+						<Tooltip text="Inner tooltip">
+							<Button>Tooltip anchor</Button>
+						</Tooltip>
+					</Tooltip>
+				</Tooltip>
+			);
+
+			// Hover the anchor. Only the outer tooltip should show.
+			await hover(
+				screen.getByRole( 'button', {
+					name: 'Tooltip anchor',
+				} )
+			);
+
+			await waitFor( () =>
+				expect(
+					screen.getByRole( 'tooltip', { name: 'Outer tooltip' } )
+				).toBeVisible()
+			);
+			expect(
+				screen.queryByRole( 'tooltip', { name: 'Middle tooltip' } )
+			).not.toBeInTheDocument();
+			expect(
+				screen.queryByRole( 'tooltip', { name: 'Inner tooltip' } )
+			).not.toBeInTheDocument();
+			expect(
+				screen.getByRole( 'button', {
+					description: 'Outer tooltip',
+				} )
+			).toBeInTheDocument();
```

Comment:
Did you mean `toBeVisible()`?

```suggestion
			).toBeVisible();
```

Reply:
[`697b30b` (#57202)](https://github.com/WordPress/gutenberg/pull/57202/commits/697b30b3a841b4f151f2e5a5c3c97b2cf8c25b3a)